### DISCUSSION TITLE
How to specify the channel when adding a package in CLI?

### DISCUSSION THREAD

**Discussion by ianzone** [prefix-dev/pixi#2010] [Q&A] [answered]
I understand that I could specify the channel in `pixi.toml`, but can I execute like 
`pixi add pytorch -c 'pytorch' cuda -c 'nvidia'`? Since my dependency list was empty at the beginning.

**Comment by synapticarbors** [comment]
I believe that there isn't an option to do that currently. You would have to `pixi project channel add <channel>` for each channel first  before adding the packages if you want to go through the cli.

**Reply by ruben-arts** [reply, marked as answer]
Correct @synapticarbors  the equivalent would be:
```
pixi project channel add nvidia pytorch
pixi add pytorch::pytorch nvidia::cuda
```
